src/features/place-bid/PlaceBidModal/PlaceBidModal.tsx

```diff
@@ -25,7 +25,7 @@ export const PlaceBidModal: FC<PlaceBidModalProps> = ({
 		setIsModalOpen(false);
 	};
 
-	const content = account ? (
+	const content = !account ? (
```

Comment:
Is `account` correct here (rather than `!account`)? Looks like the left side of the ternary shows `PlaceBidForm` and the right side shows `ConnectWallet`

Reply:
oops weird yeah that's return the opposite of expected. Updated, thanks!